dist/api_data.py

```diff
@@ -115,9 +115,21 @@ def __ge__(self, other):
     Config('checkpoint_meta', '', r'''
         the checkpoint metadata from which to start (or restart) the node''',
         undoc=True),
+    Config('flatten_leaf_page_delta', 'false', r'''
+        When enabled, page read rewrites the leaf pages with deltas to a new
+        disk image if successful''',
+        type='boolean', undoc=True),
+    Config('internal_page_delta', 'true', r'''
```

Comment:
Should the default values for this and `leaf_page_delta` be changed to `false` before merging?

Reply:
I don't think so. They'll only apply when disaggregated storage is in use, and it seems a reasonable choice for that use case.